core/src/logging/logger.rs

```diff
@@ -1,101 +0,0 @@
-//! Abscissa logger which displays through the terminal subsystem
-// TODO(tarcieri): logfile support?
-
-use super::config::Config;
-use crate::terminal::stream::{STDERR, STDOUT};
-use log::{Level, LevelFilter, Log, Metadata, Record};
-use std::io::{Error, Write};
-use termcolor::{Color, ColorSpec, StandardStreamLock, WriteColor};
-
-/// Initialize the global logger.
-///
-/// Panics if called more than once.
-pub(super) fn init(config: &Config) {
-    log::set_boxed_logger(Box::new(Logger::new(config))).expect("error configuring global logger");
-    log::set_max_level(config.level_filter);
-}
-
-/// Abscissa component for initializing the logging subsystem
-#[derive(Debug)]
-pub struct Logger {
-    level: LevelFilter,
-}
-
-impl Logger {
-    /// Create a new logger at the given log level
-    pub fn new(config: &Config) -> Self {
-        Logger {
-            level: config.level_filter,
-        }
-    }
-
-    /// Attempt to log
-    pub fn try_log(&self, record: &Record<'_>) -> Result<(), Error> {
-        let mut stream = self.level_stream(record);
-        let now = chrono::Utc::now();
```

Comment:
It looks like the removed logging code always used UTC timestamps. Should we configure `tracing`'s timestamps to also use UTC?

Reply:
I'm fine with using the system timezone (which for all of my servers is UTC) if that's the default.